10x_Experimental_Scenarios.md

```diff
@@ -1,189 +1,172 @@
+This spreadsheet shows four biological samples that are being hashed into a single loading sample (`H1`). Each sample is labeled with a different HTO (e.g. `HTO-1`) and share a common list of `Library Features` (e.g. all ADTs and HTOs used in the loading sample `H1`). The `Loading Sample` column is used to link these samples to the `Library-Level Spreadsheet` - find all libraries derived from the loading sample `H1`. 
 
 ## 2. FASTQ-Level Spreadsheet
-This is the Seq-Run-FASTQ Set level spreadsheet (see [Glossary]) that 10x techs will use to keep track of FASTQs produced from a sequencing run of a given pooled library. 
+| FASTQ | Library  | Loading Sample | 10x Lane | Library Type | Hashed Sample | Sample Index | Pooled Library | BCL | To Output | Processing Run |
+|---|---|---|---|---|---|---|---|---|---|---|
+| FQ1-GEX | L1-GEX | H1 | XL1  | GEX  | True  | SI-GA-A3 | PL1 | **BCL1** | FBM1 | PR1 |
+| FQ1-ADT | L1-ADT | H1 | XL1  | ADT  | True  |  RPI1    | PL1 | **BCL1** | FBM1 | PR1 |
+| FQ1-HTO | L1-HTO | H1 | XL1  | HTO  | True  |  D7001   | PL1 | **BCL1** | FBM1 | PR1 |
 
-| FASTQs  | Loading Sample | Sample Index | Hashed Sample | 10x Lane ID | Library Type | BCL Run ID  | Processing Run  | Library Features |
-|---|---|---|---|---|---|---|---|---|
-| H1_XL-1_BCL-1_GEX | H1 | SI-GA-A3 | True  | XL-1  | GEX  | BCL-1 | CR-1 | LF-1 |
-| H1_XL-1_BCL-1_ADT | H1 | RPI-1    | True  | XL-1  | ADT  | BCL-1 | CR-1 | LF-1 | 
-| H1_XL-1_BCL-1_HTO | H1 | D700-1   | True  | XL-1  | HTO  | BCL-1 | CR-1 | LF-1 |
 
-### Columns
 - `FASTQs`: name of the Seq-Run-FASTQ Set that is the result of a single sequencing run.
-  - Composed of the `Loading Sample` name, the `10x Lane ID`, the `BCL Run ID`, and `Library Type`
-  - Tracking the `BCL Run ID` allows us to handle the common scenario where the same sequencing pool (e.g. tube of liquid) is sequenced more than once (more than one aliquot is taken from the tube and run on the sequencer).
-- `Loading Sample`: name of the sample that is loaded into a 10x chip lane (can consist of several biolofical samples via hashing).
-- `Sample Index`: the index that is used to label the sequencing library when pooling the library into a pooled library. 10x GEX libraries have index names like `S1-GA-A3` (4 different oligos); ADT have `RPI` (single 6bp oligo), and HTO have `D700` (single 8bp oligo)
-- `Hashed Sample`: True/False, indicates whether hashing has been done (will be redundant with the `Loading Sample` naming convention).
-- `10x Lane ID`: lane number a sample is loaded into
-  - necessary for situations where the same sample is loaded into several lanes (to measure more cells from a sample)
-  - becomes part of the FASTQs' names
+  - Name includes: `Loading Sample`, `10x Lane ID`, `BCL Run ID`, `Library Type` (our examples leave off redudant information in the FASTQ names)
```

Comment:
I agree, I was just experimenting with the simplest names for each scenario. For instance, if each sample is run in just one lane I was omitting the lane number from the name. Or if one loading sample became a single GEX library, I omitted the GEX library. 

This needs to be improved though. In today's meeting we should be able to lock in our naming conventions since we have all parties in attendance.